Referentiedata.csv

```diff
@@ -1731,7 +1758,7 @@ RENTESOORT;VAR;Variabel;;;;;Financiën
 RENTESOORT;VST;Vast;;;;;Financiën
 RUIMTEDETAILSOORT;ATR;Atrium / Patio;Niet overdekt atrium of patio;24-4-2023;;RUIMTESOORT.BTR;Vastgoed
 RUIMTEDETAILSOORT;ATU;Achtertuin;Buitenruimte: tuim gelegen aan de achterzijde van de woning. Deze waarde kan gebruikt worden voor de woningwaardering indien deze duiding van tuin bekend is.;24-4-2023;;RUIMTESOORT.BTR;Vastgoed
-RUIMTEDETAILSOORT;BAD;Badkamer;Vertrek: voor een sanitaire ruimte in een woning, dat wil zeggen een ruimte die speciaal is ingericht voor lichaamsverzorging.;24-4-2023;;RUIMTESOORT.VTK;Vastgoed
+RUIMTEDETAILSOORT;BAD;Badkamer;Vertrek: voor een sanitaire ruimte in een woning, dat wil zeggen een ruimte die speciaal is ingericht voor lichaamsverzorging. EEen badkamer moet voldoen aand: en waterdichte vloerafwerking, De ruimte heeft over ten minste 50% van de oppervlakte een vrije hoogte van 2,00 m (gemeten vanaf de vloer tot het zichtbare plafond), Waterdichte afwerking tot 1,50 m hoogte voor badruimte en 1,80 m voor doucheruimte, Een wastafel inclusief (tweehands-)mengkraan en een spiegel, Een douche of bad met aansluitpunten voor warm en koud water (niet zijnde een warmwaterapparaat) en voorzien van een warm- en koudwaterkraan of een mengkraan. Een bad in een vertrek met een niet-waterdichte vloer wordt door de Huurcommissie wel gewaardeerd, omdat het bad zelf als een waterdichte afwerking wordt gezien.;24-4-2023;;RUIMTESOORT.VTK;Vastgoed
```

Comment:
Waarom is voor dit element specifiek wel de gehele (ik vermoed WWS definitie) opgenomen? Hiermee sluit je wel heel hard het gebruik uit van deze detailsoort voor andere doeleinden (die we nu per definitie niet goed apart ondervangen).

Reply:
besproken dat we de WWS volgen